pypeit/core/wavecal/defs.py

```diff
@@ -29,12 +29,47 @@ def instruments():
     return instr_dict
 
 
+class LinesBitMask(BitMask):
+    """
+    Bits for arc lines
+    """
+    version = '1.0.0'
+
+    def __init__(self):
+        mask = dict([
+            ('ArI', 'Argon I'),
+            ('HgI', 'Comment'),
+            ('KrI', 'Comment'),
+            ('NeI', 'Comment'),
+            ('XeI', 'Comment'),
+            ('CdI', 'Comment'),
+            ('ZnI', 'Comment'),
+            ('HeI', 'Comment'),
+            ('FeI', 'Comment'),
+            ('ThAr', 'Comment'),
+            ('OH', 'Comment'),
+            ('UNKNWN', 'Comment'),
+            #('OH_R24000', 'Comment'),
+            #('OH_triplespec', 'Comment'),
+            ('CuI', 'Comment'),
+            ('ArII', 'Comment'),
+            #('OH_XSHOOTER', 'Comment'),
+            #('OH_GNIRS', 'Comment'),
+            #('OH_NIRES', 'Comment'),
+            #('OH_GMOS', 'Comment'),
+            #('OH_MODS', 'Comment'),
+            #('OH_FIRE_Echelle', 'Comment'),
+            #('Ar_IR_GNIRS', 'Comment'),
+            ('Ar', 'This is for Ar_IR_GNIRS which should specify the real ion'),
```

Comment:
Perhaps this should be `'Ar_GNIRS'` or something like that? It seems like `Ar_IR_GNIRS` is commented out, which is referenced in the comment string.

Also, is this allowed to be different from the `lines()` definition below?

Reply:
We can do that if we update the Ar_IR_GNIRS line list.  Wasn't in the mood..

Reply:
I have to admit that I've lost sight as to what `lines()` is for..